blueocean/pom.xml

```diff
@@ -83,6 +83,16 @@
             <artifactId>blueocean-bitbucket-pipeline</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>blueocean-pipeline-editor</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>blueocean-jira</artifactId>
```

Comment:
Maybe make this an optional dep?

Reply:
Why is that?

Reply:
So it is optional? :)

Reply:
there is no optional dep in an aggregator - the whole point is that it chooses what is installed.

Reply:
it could be ommitted and things still work IIUC. So it is "included" but could be "removed".

Reply:
@vivek you know that with this we are still introducing the dependency of the jira plugin right?  When you install Blueocean you will always install the jira plugin. Wouldn't it make sense to leave it out of "blueocean" aggregator and let the user choose if he wants to use jira or not?

Reply:
More of product question but last I discussed about this with @i386, jira integration is mandatory feature as part of blueocean experience.

Reply:
Ok, thanks. Just wanted to be sure.